app/mailers/comment_mailer.rb

```diff
@@ -8,14 +8,14 @@ class CommentMailer < ActionMailer::Base
   # CommentMailer.notify_of_comment(user,self).deliver_now
   def notify(user, comment)
     @user = user
-    @comment = comment
```

Comment:
You need to keep this line for the functionality to work. Basically, in Rails, we can pass class variables like `@comment` to our mailer templates where they are used. If you don't define `@comment`, the template will call methods on a nil variable and it will malfunction. Can you add this line? Thanks :+1: